### PR TITLE
[python] Allow specification of X names on ingest [release-1.5]

### PR DESCRIPTION
**Pull request by johnkerl**
**Issue and/or context:** #1993 for the `release-1.5` branch

**Changes:**

**Notes for Reviewer:**

### CODE REVIEW

**Comment by johnkerl** [comment]
@aaronwolen ping 🙏

File: apis/python/src/tiledbsoma/io/ingest.py [hunk 1]
```diff
@@ -272,6 +274,14 @@ def from_h5ad(
         platform_config: Platform-specific options used to create this array, provided in the form
         ``{"tiledb": {"create": {"sparse_nd_array_dim_zstd_level": 7}}}`` nested keys.
 
+        obs_id_name: Which AnnData ``obs`` column to use for append mode.
+
+        var_id_name: Which AnnData ``var`` column to use for append mode.
```
Comment (aaronwolen) [inline apis/python/src/tiledbsoma/io/ingest.py:279]: Thanks for documenting these! A little more info on how they're used for appending might out help users. I think these args are also used to rename the index of incoming obs/var dataframes prior to ingest, right?

**Review by aaronwolen — APPROVED** [review]
One minor comment re docs but otherwise looks perfect!

**Comment by johnkerl** [comment]
@aaronwolen thank you for your review!

https://github.com/single-cell-data/TileDB-SOMA/pull/1995/commits/f3963c5ab2d12b85867c18bc2fc96b541e4e0ff7 offers a bit more detail -- however, I'm not sure what you want -- if you could state what you want (an example sentence perhaps) that would be helpful -- thank you!

**Comment by aaronwolen** [comment]
Just noting we discussed and resolved on a call.

**Pull request merged**